src/components/Game/GameView.tsx

```diff
@@ -222,7 +212,7 @@ const GameView: React.FC<Props> = ({
       );
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [keyboardDimension, currentMapping]);
+  }, [keyboardDimension, indexToNotesMap]);
```

Comment:
```suggestion
  }, [keyboardDimension, delayedStartTime]);
```

Here should 'delayedStartTime' be a dependency?

Reply:
Yup. Thanks for pointing it out 😅